community/graphdb-api/src/main/java/org/neo4j/graphdb/spatial/impl/Point.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.graphdb.spatial.impl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import org.neo4j.graphdb.spatial.CRS;
+import org.neo4j.graphdb.spatial.Coordinate;
+
+/**
+ * Default implementation of a {@link org.neo4j.graphdb.spatial.Point}
+ */
+public final class Point implements org.neo4j.graphdb.spatial.Point
```

Comment:
Why does the API definition module contain a "default implementation" - who is it for?

Reply:
See above. Also, all downstream repositories like docs, jmh-benchmarks,... which constantly break because of using internal types instead of public API. So let's change that

Reply:
I previously added Point implementations to the documentation module so that it would not use PointValue.

Reply:
I prefer all downstream modules used the same instead of coding their own.